src/python/setup.py.in

```diff
@@ -55,6 +56,20 @@ ext_modules.append(Extension(
     extra_compile_args=extra_compile_args + [@GUDHI_PYBIND11_EXTRA_COMPILE_ARGS@],
     ))
 
+if "bottleneck" in pybind11_modules:
```

Comment:
I think we could manage hera and bottleneck by doing a loop on each module from pybind11_modules, no ?
This would help to go with pybind11 for other python modules.

Reply:
That was my first thought, but hera wants some extra includes, which complicates things a bit.